src/mergeSortLoop.js

```diff
@@ -0,0 +1,42 @@
+export default function mergeSort(toSort){
+  let currentLevel = []
+  toSort.map( function(number){
+    let subArray = []
+    subArray.push(number)
+    currentLevel.push(subArray)
+  })
+  let nextLevel = []
+
+  while(currentLevel.length > 1){
```

Comment:
There's a lot going on in this loop - consider refactoring to pull out logic that can exist in isolation (for example, the swapping of elements between sub arrays could be placed in a separate function).